odoo/addons/base/module/module.py

```diff
@@ -283,7 +283,7 @@ def _get_icon_image(self):
         ('GPL-3 or any later version', 'GPL-3 or later version'),
         ('AGPL-3', 'Affero GPL-3'),
         ('LGPL-3', 'LGPL Version 3'),
-        ('Other OSI approved licence', 'Other OSI Approved Licence'),
+        ('Other OSI approved license', 'Other OSI Approved License'),
```

Comment:
Please change only the label, not the selection. We use that value in different integrations like the app store.